rpc/block_svc_routines.c

```diff
@@ -4375,6 +4375,23 @@ block_create_common(blockCreate *blk, char *control, char *volServer, char *prio
   }
   reply->exit = -1;
 
+  if (GB_ALLOC_N(reply->out, 8192) < 0) {
+    goto out;
+  }
+
+  if (GB_ASPRINTF(&exec, "grep -cs 'name.*%s' /etc/target/saveconfig.json /etc/target/saveconfig.json.temp",
```

Comment:
@lxbsz lets use GB_SAVECONFIG_CHECK here ?

Reply:
I mean old one

Reply:
I am afraid it is not appropriated.
As I descripted above, each time the GBID will change, so we need to check the NAME instead.

Reply:
@lxbsz 

That is why I said old version of GB_SAVECONFIG_CHECK :-)

The problem I see with this is, your regular expression will match all below cases say if your block name is 'block1'
```
# cat ok.txt
blockname_1isblock1                                                              
blockname_1isblock123                                                            
blockname_1isblock1x123                                                          
name_isblock1kl                                                                  
myblockname1block1

# grep  'name.*block1' ok.txt 
blockname_1isblock1
blockname_1isblock123
blockname_1isblock1x123
name_isblock1kl
myblockname1block1

```

Hence lets use what we did with GB_SAVECONFIG_CHECK earlier

```
# define GB_SAVECONFIG_CHECK_BVN "grep -m 1 '\"name\": \"%s\",' " GB_SAVECONFIG " " GB_SAVECONFIG ".temp > " DEVNULLPATH
```

BVN = block volume name

Reply:
Cool, this make sense.
I will fix it.